assets/js/app.js

```diff
@@ -40,6 +40,12 @@ let calendar = new Calendar(calendarEl, {
       dotEl.style.display = "none";
     }
   },
+  eventClick: function(event) {
+    if (event.event.url) {
+      event.jsEvent.preventDefault();
+      window.open(event.event.url, "_blank");
+    }
+  },
```

Comment:
@TORIFUKUKaiou (日本語で書きます😅

ありがとうございます！2点、修正入れたいです 👀

- [公式ドキュメント](https://fullcalendar.io/docs/eventClick)の命名にあわせる
- noopenerをつける
  - 参考: https://www.1st-net.jp/blog/target-blank-noopener/

```suggestion
  eventClick: function(info) {
    info.jsEvent.preventDefault();
    if (info.event.url) {
      window.open(info.event.url, "_blank", "noopener");
    }
  }
```